astropy/io/votable/tests/table_test.py

```diff
@@ -14,6 +14,8 @@
 from astropy.io.votable.table import parse, writeto
 from astropy.io.votable import tree, conf
 from astropy.io.votable.exceptions import VOWarning, W39, E25
+from astropy.table import Column, Table
+from astropy.units import UnrecognizedUnit
```

Comment:
I wonder why these weren't top level before. Perhaps we used to have circular import problem? Anyways, CI passed, so this should probably be okay... 🤞